src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedCommunityCareLocation.jsx

```diff
@@ -13,7 +13,6 @@ export default function ConfirmedCommunityCareLocation({ appointment }) {
   if (!location.address) {
     return (
       <>
-        <h4 className="vaos-appts__block-label">{location.name}</h4>
```

Comment:
Is this always empty? If it's not always empty, I think we'd want to make it conditional on whether `location.name` existed or not, to avoid the axe error.

Reply:
adding a check if `location.name` exists where the Community Care appt is store in Vista